src/Adapter/SqlManager/CommandHandler/DeleteSqlRequestHandler.php

```diff
@@ -0,0 +1,78 @@
+<?php
+/**
+ * 2007-2018 PrestaShop.
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2018 PrestaShop SA
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace PrestaShop\PrestaShop\Adapter\SqlManager\CommandHandler;
+
+use PrestaShop\PrestaShop\Core\Domain\SqlManagement\Command\DeleteSqlRequestCommand;
+use PrestaShop\PrestaShop\Core\Domain\SqlManagement\CommandHandler\DeleteSqlRequestHandlerInterface;
+use PrestaShop\PrestaShop\Core\Domain\SqlManagement\Exception\CannotDeleteSqlRequestException;
+use PrestaShop\PrestaShop\Core\Domain\SqlManagement\Exception\SqlRequestException;
+use PrestaShop\PrestaShop\Core\Domain\SqlManagement\Exception\SqlRequestNotFoundException;
+use PrestaShopException;
+use RequestSql;
+
+/**
+ * Class DeleteSqlRequestHandler.
+ *
+ * @internal
+ */
+final class DeleteSqlRequestHandler implements DeleteSqlRequestHandlerInterface
+{
+    /**
+     * {@inheritdoc}
+     *
+     * @throws CannotDeleteSqlRequestException
+     * @throws SqlRequestNotFoundException
+     * @throws SqlRequestException
+     */
+    public function handle(DeleteSqlRequestCommand $command)
+    {
+        $entityId = $command->getSqlRequestId()->getValue();
+
+        try {
+            $entity = new RequestSql($entityId);
+
+            if (0 >= $entity->id) {
+                throw new SqlRequestNotFoundException(
+                    sprintf('SqlRequest with id "%s" was not found for edit', var_export($entityId, true))
+                );
+            }
+
+            if (false === $entity->delete()) {
+                throw new CannotDeleteSqlRequestException(
+                    sprintf('Could not delete SqlRequest with id %s', var_export($entityId)),
```

Comment:
@sarjon isn't there a missing `true` in this `var_export` call?

Reply:
you are totally right @jolelievre